### ISSUE TITLE
Faction tags recommendation

### ISSUE THREAD

**Issue by Tincon1** [DRE2N/FactionsOne#45]
Hey, would it be possible to have faction tags like the old factions?
Where as if allied to a faction you see their tag in purple instead of white
and whereas if enemied you see them in red
and for your own faction you see it in yellow?

Sorry for putting that in rating, I updated my rating!

**Comment by Sataniel98** [comment]
Hi, I recommend using the FactionChat plugin for best results.

https://dev.bukkit.org/projects/factionchat

**Comment by Tincon1** [comment]
In main chat, the tags are just white? Can it be yellow for people in theyr own faction, when a ally types their tag is purple, enemy red? like regular factions was

**Comment by Tssua** [comment]
yeah I agree this plugin works really well except the fact that the faction tag stays white, can you make it like the old faction plugins worked? when typing in chat your faction shows up in yellow, allies show up in purple, and enemies show up in red that would be a huge help!

**Comment by Sataniel98** [comment]
First of all, this _is_ the old Factions, and everything should work exactly as it used to. I haven't added or removed any chat features. Did you set parseTagsColored to true in the config?
